cluster-baseline-settings/aad-pod-identity/aad-pod-identity.yaml

```diff
@@ -201,32 +204,25 @@ spec:
     spec:
       serviceAccountName: aad-pod-identity-nmi
       hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
       volumes:
       - hostPath:
           path: /run/xtables.lock
           type: FileOrCreate
         name: iptableslock
       containers:
       - name: nmi
-        image: "mcr.microsoft.com/k8s/aad-pod-identity/nmi:1.6.1"
+        image: "mcr.microsoft.com/oss/azure/aad-pod-identity/nmi:v1.6.3"
```

Comment:
should this be also `imported`?

Reply:
if yes, but since this is temporary `yaml` we could leave as-is.

Reply:
Nope, doesn't need to be "mcr" is a "safe" public repo to pull from.  Doesn't throttle.  And, since we'll be moving to AAD as an add-on soon enough, it'll go away as you said.